.github/workflows/e2e-tests.yml

```
@@ -167,4 +167,33 @@ jobs:
           event_type: e2e
           report_name: ${{needs.create-test-matrix.outputs.runName}}
 
+  slack-notification:
+    name: "Slack notification"
+    runs-on: ubuntu-latest
+    if: ${{ ! cancelled() }}
+    needs: [e2e-tests]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up tools
+        uses: ./.github/actions/tool-setup
+
+      # remove this step after action release and replace with the action from marketplace
+      - name: Build action
+        env:
+          COMPOSER_ROOT_VERSION: dev-trunk
+        run: |
+          pnpm install
+          composer install
+          composer build-development
+        working-directory: ./projects/github-actions/test-results-to-slack
+
+      - name: "Send notification"
+        uses: ./projects/github-actions/test-results-to-slack # replace with the action from marketplace after the action is released
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          slack_token: ${{ secrets.SLACK_TOKEN }}
+          slack_channel: ${{ secrets.SLACK_E2E_CHANNEL }}
```

Should we name this differently, such as `slack_e2e_channel`?

```suggestion
          slack_e2e_channel: ${{ secrets.SLACK_E2E_CHANNEL }}
```

---

This is intentional, because in the action's context, there is only a Slack channel. It can be for e2e, unit, or any other type of tests.